framework/Models/PostProcessors/BasicStatistics.py

```diff
@@ -998,21 +1017,17 @@ def __runLocal(self, inputData):
         en = targWeight.sum()**2/np.sum(targWeight**2)
```

Comment:
In our code, we are using equivalentSamples, which are precomputed. We may merge equivalentSamples and self.sampleSize in the future.  Could you update this line? the following is an example:
```
    metric = 'sigma'
    if len(needed[metric]['targets'])>0:
      self.raiseADebug('Starting calculate standard error on "'+metric+'"...')
      varList = list(needed[metric]['targets'])
      if self.pbPresent:
        en = calculations['equivalentSamples'][varList]
        factor = 2.0 * (en - 1.0)
        factor = self._computePower(0.5,factor)
      else:
        factor = np.sqrt(2.0 * (float(self.sampleSize) - 1.0))
      calculations[metric+'_ste'] = calculations['sigma'][varList] / factor
```

Reply:
I did not realize this was available. I'll make the change.